### PR TITLE
[invite] Implement GitHub API interface methods

### PR DESCRIPTION
**Pull request by rcebulko**
```
GitHub interface
  inviteUser
    ✓ PUTs to /orgs/:org/memberships/:username (19ms)
    ✓ returns true when the user is invited (2ms)
    ✓ returns false when the user is already a member (1ms)
  addComment
    ✓ POSTs comment to /repos/:owner/:repo/issues/:issue_number/comments (2ms)
  assignIssue
    ✓ POSTs assignee to /repos/:owner/:repo/issues/:issue_number/assignees (1ms)
```

### CODE REVIEW

File: invite/test/github.test.ts [hunk 1]
```diff
@@ -48,35 +48,6 @@ describe.skip('GitHub interface', () => {
     }
   });
 
-  describe('userIsMember', () => {
```
Comment (danielrozenberg) [inline invite/test/github.test.ts]: Why did you remove these tests?

File: invite/src/github.ts [hunk 2]
```diff
@@ -62,8 +55,12 @@ export class GitHub {
     issue_number: number,
     comment: string
   ): Promise<void> {
-    // https://octokit.github.io/rest.js/#octokit-routes-issues-create-comment
-    // octokit.issues.createComment({owner, repo, issue_number, body})
+    await this.client.issues.createComment({
```
Comment (danielrozenberg) [inline invite/src/github.ts]: Why would this bot create comments? (genuine q)

File: invite/test/github.test.ts (hunk 1 above)
Comment (rcebulko) [reply invite/test/github.test.ts]: The send-invite API endpoint responds with a field indicating whether the invite was sent (state: pending) or the user was already a member (state: active). So, there's no point to checking if the user is a member, we can just send an invite and GitHub will ignore it if they're a member.

When the thread on limiting invites to org members or group members is resolved, this may be added for that check, but for now it can go :wastebasket:

File: invite/src/github.ts (hunk 2 above)
Comment (rcebulko) [reply invite/src/github.ts]: To update threads that an invite was sent/accepted, or in the event of an error that the invite was not sent because of an issue. Without some kinda feedback in the form of a comment, there's no indication that the bot actually did anything.
